src/main/java/com/joona/rest/service/RequestValidator.java

```diff
@@ -0,0 +1,73 @@
+package com.joona.rest.service;
```

Comment:
Request Validator is not meant to be a "service" ... please move this to a different package ...under "request"